var/spack/repos/builtin/packages/py-flawfinder/package.py

```diff
@@ -0,0 +1,28 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyFlawfinder(PythonPackage):
+    """a program that examines source code looking for security weaknesses"""
+
+    homepage = "http://dwheeler.com/flawfinder/"
+
+    version('2.0.19', sha256='f18f0021d9a2f3ffc1c1e32aa7bb1bb5ca1bad91b82b32157f44e5d0a1a23ad0',
```

Comment:
Looks like it's also available at http://dwheeler.com/flawfinder/flawfinder-2.0.19.tar.gz, might be better to build from source